package-dev/Runtime/SentryInitialization.cs

```diff
@@ -3,6 +3,8 @@
 #define SENTRY_NATIVE_IOS
 #elif UNITY_ANDROID
 #define SENTRY_NATIVE_ANDROID
+#elif UNITY_STANDALONE_WIN && ENABLE_IL2CPP
```

Comment:
```suggestion
#elif UNITY_STANDALONE_WIN
```
Is IL2CPP a requirement for native support? What about native plugins on mono?

Reply:
@bitsandfoxes we'd be taking the signal handler from Mono so I believe that's not what we want

Reply:
Also makes me realize we need to be clear about minidump support requiring IL2CPP

Reply:
I've also added a warning to the PostBuild to warn the user when he has the Native SDK support enabled in settings but then uses non-IL2CPP backend.